rosdep/base.yaml

```
@@ -2982,6 +2982,12 @@ libqwt6:
   fedora: [qwt-devel]
   gentoo: ['x11-libs/qwt:6']
   ubuntu: [libqwt-dev]
+libqwt6-qt5-dev:
+  arch: [qwt]
+  debian: [libqwt-qt5-dev]
+  fedora: [qwt-qt5-devel]
+  gentoo: ['x11-libs/qwt:6']
+  ubuntu: [libqwt-qt5-dev]
```

```suggestion
  ubuntu:
    '*': [libqwt-qt5-dev]
    trusty: null
```

This will say that trusty isn't supported and otherwise all the other platforms are EOL already.

---

Wow, that's a neat trick! I had no idea that the wildcard model could be inverted like this.